examples/test.py

```diff
@@ -0,0 +1,55 @@
+
+# import os
+# from tvb_epilepsy.base.constants.configurations import FOLDER_VEP_HOME
+# from tvb_epilepsy.base.h5_model import convert_to_h5_model, read_h5_model
+# from tvb_epilepsy.service.model_inversion.stan.cmdstan_service import CmdStanService
+# from matplotlib import pyplot
+
+from tvb_epilepsy.service.stochastic_parameter_factory import set_parameter
+
+if __name__ == "__main__":
+
+    K = set_parameter("K", optimize_pdf=True, use="manual", K_lo=0.0, K_hi=10, K_pdf="lognormal",
```

Comment:
This is just a dummy function I am using whenever I need to test some code, since pycharm is not as easy to use as matlab in that respect. We can delete it and I will recreate it, if needed, in the review_DP branch that I should start working in..